src/parser/jobs/blm/modules/Gauge.js

```diff
@@ -344,28 +362,28 @@ export default class Gauge extends Module {
 			}))
 		}
 
-		if (this._lostFoul) {
+		if (this._lostPolyglot) {
 			this.suggestions.add(new Suggestion({
-				icon: ACTIONS.FOUL.icon,
-				content: <Trans id="blm.gauge.suggestions.lost-foul.content">
-					You lost <ActionLink {...ACTIONS.FOUL}/> due to dropped <ActionLink {...ACTIONS.ENOCHIAN}/>. <ActionLink {...ACTIONS.FOUL}/> is your strongest GCD, so always maximize its casts.
+				icon: ACTIONS.XENOGLOSSY.icon,
+				content: <Trans id="blm.gauge.suggestions.lost-polyglot.content">
+					You lost <ActionLink {...ACTIONS.XENOGLOSSY}/> due to dropped <ActionLink {...ACTIONS.ENOCHIAN}/>. <ActionLink {...ACTIONS.XENOGLOSSY}/> is your strongest GCD, so always maximize its casts.
```

Comment:
We should probably have this worded more general as "Polyglot actions" instead of Xeno. Both will be used depending on aoe or sinle target, and I don't wanna have people think they should only use one of them all the time.

`You lost <ActionLink {...ACTIONS.XENOGLOSSY}/> or <ActionLink {...ACTIONS.FOUL}/> due to dropped...`

Might be an idea as well.

Reply:
Redid the wording to try to get at the "Spend something and don't drop or overwrite stacks" point without pushing one or the other (besides switching the icon for the suggestion to be the Xenoglossy one instead). Lemme know if you've got other ideas.